test/e2e/fixtures/then.go

```diff
@@ -86,9 +87,15 @@ func (t *Then) ExpectWorkflowNode(selector func(status wfv1.NodeStatus) bool, f
 		if n != nil {
 			_, _ = fmt.Println("Found node", "id="+n.ID, "type="+n.Type)
 			if n.Type == wfv1.NodeTypePod {
+				version := util.PodNameV1
+				annotations := metadata.GetAnnotations()
+				if annotations[common.AnnotationKeyPodNameVersion] == util.PodNameV2.String() {
```

Comment:
Reuse GetWorkflowPodNameVersion?

Reply:
I was thinking about it but didn't have a workflow object. Would you recommend building one using the metadata variable, or just leaving as is?

Reply:
I think creating a workflow object would be better to reduce code redundancy and make it easier later when we start removing v1 pod name.

Reply:
Good call. I'll make the switch